bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -232,10 +232,27 @@ public Row readModifyWriteRow(ReadModifyWriteRowRequest request) {
   private ResultScanner<Row> streamRows(ReadRowsRequest request) {
     boolean isGet = request.getTargetCase() == ReadRowsRequest.TargetCase.ROW_KEY;
 
+    int streamingBufferSize = retryOptions.getStreamingBufferSize();
```

Comment:
Do we want to initialize streamingBufferSize here? we don't for batchRequestSize and channel.